smartlearner/optimizers/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .optimizer import Optimizer
 from .sgd import SGD
+from .adagrad import ADAGRAD
```

Comment:
Oh!, I thought it was an acronym like SGD. In the paper, they seem to use `AdaGrad` let use that.